src/Model/Photo.php

```diff
@@ -622,12 +641,8 @@ public static function setPermissionFromBody($body, $uid, $original_contact_id,
 		}
 
 		foreach ($images as $image) {
-			if (!stristr($image, DI::baseUrl() . '/photo/')) {
-				continue;
-			}
-			$image_uri = substr($image,strrpos($image,'/') + 1);
-			$image_uri = substr($image_uri,0, strpos($image_uri,'-'));
-			if (!strlen($image_uri)) {
+			$image_rid = self::ridFromURI($image);
+			if (!$image_rid) {
```

Comment:
I'd like having `if (empty($image_rid))` , which - at least I find - more explicit (because it's not a boolean type per se..)